jobs/transforms/exp_validation_test.py

```diff
@@ -249,3 +254,267 @@ def test_validate_exp_model_object_with_invalid_state_property(self):
+                'invalid is not allowed')
+        ])
+
+    def test_exploration_rights_change_object_with_invalid_status(
+            self):
+        invalid_commit_cmd_model = (
+            exp_models.ExplorationRightsSnapshotMetadataModel(
+                id='model_id-1',
+                created_on=self.YEAR_AGO,
+                last_updated=self.NOW,
+                committer_id='committer_id',
+                commit_type='create',
+                commit_cmds=[{
+                    'cmd': 'change_exploration_status',
+                    'old_status': rights_domain.ACTIVITY_STATUS_PRIVATE,
+                    'new_status': 'invalid'
+                }])
+        )
+
+        output = (
+            self.pipeline
+            | beam.Create([invalid_commit_cmd_model])
+            | beam.ParDo(
+                exp_validation.ValidateExplorationRightsSnapshotMetadataModel())
+        )
+
+        self.assert_pcoll_equal(output, [
+            base_validation_errors.CommitCmdsValidateError(
+                invalid_commit_cmd_model,
+                {
+                    'cmd': 'change_exploration_status',
+                    'old_status': rights_domain.ACTIVITY_STATUS_PRIVATE,
+                    'new_status': 'invalid'
+                },
+                'Value for new_status in cmd change_exploration_status: '
+                'invalid is not allowed')
+        ])
+
+
+class ValidateExplorationCommitLogEntryModelTests(
+        job_test_utils.PipelinedTestBase):
+
+    def test_validate_rights_model(self):
+        invalid_commit_cmd_model = exp_models.ExplorationCommitLogEntryModel(
+            id='rights_id123',
+            created_on=self.YEAR_AGO,
+            last_updated=self.NOW,
+            commit_type='test-type',
+            user_id='',
+            exploration_id='123',
+            post_commit_status='private',
+            commit_cmds=[{'cmd': 'create_new'}])
+
+        output = (
+            self.pipeline
+            | beam.Create([invalid_commit_cmd_model])
+            | beam.ParDo(
+                exp_validation.ValidateExplorationCommitLogEntryModel())
+        )
+
+        self.assert_pcoll_equal(output, [])
+
+    def test_validate_exploration_model(self):
+        invalid_commit_cmd_model = exp_models.ExplorationCommitLogEntryModel(
+            id='exploration_id123',
+            created_on=self.YEAR_AGO,
+            last_updated=self.NOW,
+            commit_type='test-type',
+            user_id='',
+            exploration_id='123',
+            post_commit_status='private',
+            commit_cmds=[{
+                'cmd': base_models.VersionedModel.CMD_DELETE_COMMIT}])
+
+        output = (
+            self.pipeline
+            | beam.Create([invalid_commit_cmd_model])
+            | beam.ParDo(
+                exp_validation.ValidateExplorationCommitLogEntryModel())
+        )
+
+        self.assert_pcoll_equal(output, [])
+
+    def test_validate_other_model(self):
```

Comment:
Please change the name of the test to something appropriate, like test_raises_commit_cmd_none_error.

Reply:
Done